sentry_sdk/integrations/logging.py

```diff
@@ -374,7 +375,10 @@ def _capture_log_from_record(client, record):
         if record.lineno:
             attrs["code.line.number"] = record.lineno
         if record.pathname:
-            attrs["code.file.path"] = record.pathname
+            if record.pathname.startswith(project_root):
```

Comment:
`project_root` might be `None` so we need to be more careful here:
```suggestion
            if project_root is not None and record.pathname.startswith(project_root):
```

Reply:
done, test added too